schema/mysql/v8/visibility/schema.sql

```diff
@@ -44,6 +44,7 @@ CREATE TABLE executions_visibility (
   TemporalSchedulePaused        BOOLEAN       GENERATED ALWAYS AS (search_attributes->"$.TemporalSchedulePaused"),
   TemporalNamespaceDivision     VARCHAR(255)  GENERATED ALWAYS AS (search_attributes->>"$.TemporalNamespaceDivision"),
   BuildIds                      JSON          GENERATED ALWAYS AS (search_attributes->"$.BuildIds"),
+  PauseActivityTypes            JSON          GENERATED ALWAYS AS (search_attributes->"$.PauseActivityTypes"),
```

Comment:
Rename: `s/PauseActivityTypes/PausedEntities`
You missed the renaming in a few places, please fix them.

Reply:
done